### ISSUE TITLE
Asia/Baku time zone change from GMT+5 to GMT+4

### ISSUE THREAD

**Issue by iskrenvankov** [ndom91/react-timezone-select#50]
Hi,

I'm using `react-timezone-select` in one of my projects and a user of ours reported an issue with the `Asia/Baku` - Baku, Tbilisi, Yerevan time zone.
It appears as (GMT+5:00) but it should be (GMT+4:00).

Digging a bit deeper, I realised that they actually literally changed their time zone recently.
The update was reflected in spacetime `v6.16.2` https://github.com/spencermountain/spacetime/releases/tag/6.16.2 and respectively in react-timezone-select `v1.1.11` https://github.com/ndom91/react-timezone-select/commit/2d945cb0b7d91f8f239965a9cd153255376fc6f1

So having seen these updates, I thought maybe my project's version was old, but that's not the case:
```
react-timezone-select@^1.1.13:
  version "1.1.13"
  resolved "https://registry.yarnpkg.com/react-timezone-select/-/react-timezone-select-1.1.13.tgz#76090b9511b14e7dbf9474b30c61bd77a64a1852"
  integrity sha512-lHcjnax3HtLyHCa+yI3xO6IOLfMI4rTjlFGHN2fBauerKi4ee+ZGoQ6zw7HHisTkWmyU4iE/Xf3gSh1pEQMW8g==
  dependencies:
    react-select "^5.0.0"
    spacetime "^6.16.3"
    timezone-soft "^1.3.1"
```

I'm a bit stumped since it seems that I'm running the latest versions of both `react-timezone-select` and `spacetime` underneath and yet the wrong time zone is displayed for `Asia/Baku`.

Any ideas why that might be?

Thanks,
Iskren

**Comment by ndom91** [comment]
Hi Iskren, sorry its taken so long for me to answer! I was the one who originally made that update PR to spacetime haha.

But oddly enough, even using an up-to-date version, I'm still seeing it return +5. Can you try `spacetime` in a codesandbox and provide a reproduction where +4 works?

**Comment by iskrenvankov** [comment]
Hi, yes actually that was one of my sanity checks back when I discovered the issue first. I went to this spacetime demo here: https://observablehq.com/@spencermountain/spacetime and just changed one of the cells to 

```
{
  return html`<div>${spacetime.version}</div><div>${spacetime.now().goto('Asia/Baku').timezone().current.offset}</div>`
}
```

which then prints out 

```
7.1.0
4
```

So we know that in spacetime 7.1.0 the offset is correct (+4).

**Comment by ndom91** [comment]
Hmm very strange, I went through my code here last night and tested to see what came back when I gave it 'Asia/Baku', and I got the TZ object, including `offset: 5`. 

I'll have to double check, maybe I oversaw something

**Comment by ndom91** [comment]
@iskrenvankov sorry this took so long! I've recently released `1.3.2` with the `spacetime` dependency update. Meaning `Baku` is finally in `offset: 4` :+1:
